config/jobs/dependency-watchdog/dependency-watchdog-kind-unit-tests.yaml

```diff
@@ -0,0 +1,77 @@
+presubmits:
+  gardener/dependency-watchdog:
+    - name: pull-dependency-watchdog-kind-unit-tests
+      cluster: gardener-prow-build
+      always_run: true
+      skip_branches:
+        - release-v\d+.\d+ # don't run on release branches for now (add a job per branch later)
+      decorate: true
+      decoration_config:
+        timeout: 60m
+        grace_period: 15m
+      labels:
+        preset-dind-enabled: "true"
+        preset-kind-volume-mounts: "true"
+      annotations:
+        description: Runs KIND cluster based unit tests for dependency watchdog developments in pull requests
+        fork-per-release: "true"
+      spec:
+        containers:
+          - image: gcr.io/k8s-staging-test-infra/krte:v20221107-33c989e684-master
+            command:
+              - wrapper.sh
+              - bash
+              - -c
+              - make kind-tests
+            # we need privileged mode in order to do docker in docker
+            securityContext:
+              privileged: true
+            resources:
+              requests:
+                cpu: 12
+                memory: 18Gi
+            env:
+              - name: SKAFFOLD_UPDATE_CHECK
+                value: "false"
+              - name: SKAFFOLD_INTERACTIVE
+                value: "false"
```

Comment:
Are you using `skaffold` for your tests? If not you could remove these envs.

Reply:
removed as we are not using skaffold